### ISSUE TITLE
`bt` shows strange information on nested block

### ISSUE THREAD

**Issue by ko1** [ruby/debug#90]
```
[master]$ exe/rdbg -e 'b 3;; c;; bt' target.rb
[1, 7] in target.rb
=>    1| 1.times{
      2|   1.times{
      3|     p 1
      4|   }
      5| }
      6|
      7| __END__
=>#0    <main> at target.rb:1
(rdbg:init) b 3
#0  BP - Line  /mnt/c/ko1/src/rb/ruby-debug/target.rb:3 (line)
(rdbg:init) c
[1, 7] in target.rb
      1| 1.times{
      2|   1.times{
=>    3|     p 1
      4|   }
      5| }
      6|
      7| __END__
=>#0    block in levels) at target.rb:3
  #1    [C] Integer#times at target.rb:2
  # and 3 frames (use `bt' command for all frames)

Stop by # BP - Line  /mnt/c/ko1/src/rb/ruby-debug/target.rb:3 (line) 0
(rdbg:init) bt
=>#0    block in levels) at target.rb:3
  #1    [C] Integer#times at target.rb:2
  #2    block in <main> at target.rb:2
  #3    [C] Integer#times at target.rb:1
  #4    <main> at target.rb:1
```

`=>#0    block in levels) at target.rb:3` seems strange.
Could you check it if you have a time? @st0012

**Comment by st0012** [comment]
@ko1 sure, checking 👍

**Comment by st0012** [comment]
@ko1 this should be addressed by #91
